citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -360,10 +362,10 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			String href = citation.hasPreferredUrl() ? citation.getCustomUrl(citation.getPreferredUrlId()) : citation.getOpenurl();
     			
     			out.println("\t\t<td headers=\"details\">");
-    			out.println("\t\t\t<a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a><br />");
-    			out.println("\t\t\t\t" + Validator.escapeHtml( citation.getCreator() ) );
-    			out.println("\t\t\t\t" + Validator.escapeHtml( citation.getSource() ) );
-    			out.println("\t\t\t<div class=\"itemAction\">");
+    			out.println("\t\t\t<div class=\"floatLeft fullWidth\"><div class=\"floatLeft\"><a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a>");
```

Comment:
Ideally CSS names should be semantic and actual layout used should be in the CSS files.